src/file-chooser.c

```diff
@@ -439,6 +439,7 @@ static XdpOptionKey open_file_options[] = {
   { "accept_label", G_VARIANT_TYPE_STRING, NULL },
   { "modal", G_VARIANT_TYPE_BOOLEAN, NULL },
   { "multiple", G_VARIANT_TYPE_BOOLEAN, NULL },
+  { "directory", G_VARIANT_TYPE_BOOLEAN, NULL },
```

Comment:
This means we'll pass the option to the backend, so we should probably update the backend interface description at the same time to also mention that option.